### ISSUE TITLE
Fails on Chinese people

### ISSUE THREAD

**Issue by f3uplift** [ageitgey/face_recognition#612]
* face_recognition version:
* Python version: 3.5.2
* Operating System: Ubuntu 16.04

### The library can't separate chinese people

I tried finding 
https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=2ahUKEwjKwbLM6JTdAhWGMewKHekXBIYQjRx6BAgBEAU&url=https%3A%2F%2Flaotiantimes.com%2F2016%2F09%2F05%2Fchinese-prime-minister-to-visit-laos%2F&psig=AOvVaw0jNXiklPM4YaxtrqIEOjQe&ust=1535719348182900
in
https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=2ahUKEwiWwOmD55TdAhVNMewKHaGnBo0QjRx6BAgBEAU&url=https%3A%2F%2Fwww.japantimes.co.jp%2Fnews%2F2015%2F01%2F09%2Fnational%2Fhistory%2Fchina-brings-sex-slave-issue-spotlight%2F&psig=AOvVaw0jNXiklPM4YaxtrqIEOjQe&ust=1535719348182900
image and it identified the first person as the target person. Thus it can't accurately compare faces.

**Comment by ageitgey** [comment]
Correct. You may have some success lowering the threshold, but there are limitations to the accuracy here. Please see this for more details: https://github.com/ageitgey/face_recognition/wiki/Face-Recognition-Accuracy-Problems#question-face-recognition-works-well-with-european-individuals-but-overall-accuracy-is-lower-with-asian-individuals
